microsoft/botframework-solutions
# [Enterprise Bot Generator][Typescript] Implement mocha

## Description

- Implement Mocha on Enterprise Bot.

## Related Issue
Fix #639

## Testing Steps

1. Open `AI\templates\Enterprise-Template\src\typescript\enterprise-bot`.
2. Run `npm i` and check that Mocha dependency was installed.

## Checklist 

~- [] I have commented my code, particularly in hard-to-understand areas~ 
~- [] I have added or updated the appropriate unit tests~ 
~- [] I have updated related documentation~ 

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant 

~- [ ] A duplicate issue is filed to track future  work.~ 

If you have updated responses or `.lu` files: 

~- [] All languages have been updated~ 
~- [X] You have tested deployment with your new models~
